src/style.css

```diff
@@ -0,0 +1,58 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  /* width: 96px;
+  height: 96px; */
+  display: flex;
+  flex-direction: column;
+  /* justify-content: space-between; */
+  /* align-items: space-between;
+  align-content: space-between; */
+  /* border: 1px solid black; */
+}
+
+.stars {
+  display: flex;
+  /* align-content: space-between; */
+  /* flex-direction: row; */
+  /* height: 100%; */
+  align-self: space-between;
+}
+
+.stars__star {
+  display: flex;
+  background: url(./images/star.svg) no-repeat;
+  background-position: center;
+  vertical-align: bottom;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  /* justify-content: space-between; */
+}
+
+.stars .stars__star:last-child {
```

Comment:
I think it would look good if you set margin for stars here (remember about :not pseudo class) ;)

Reply:
corrected